src/components/App/App.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+import './App.scss';
+
+import posts from '../../api/posts';
+import comments from '../../api/comments';
+import users from '../../api/users';
+import { PostList } from '../PostList/PostList';
+
+const preparedPosts: Post[] = posts.map(post => {
+  const relaventUser = users.find(user => user.id === post.userId) || null;
```

Comment:
```suggestion
  const relevantUser = users.find(user => user.id === post.userId) || null;
```
typo in the word `relevant`